### PR TITLE
post about go

### PR DESCRIPTION
**Pull request by jks8787**

### CODE REVIEW

File: _posts/2015-09-29-post-about-go.markdown [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+> * It is possible to compile a large Go program in a few seconds on a single computer.
+> * Go provides a model for software construction that makes dependency analysis easy and avoids much of the overhead of C-style include files and libraries.
+> * Go's type system has no hierarchy, so no time is spent defining the relationships between types. Also, although Go has static types the language attempts to make types feel lighter weight than in typical OO languages.
+> * Go is fully garbage-collected and provides fundamental support for concurrent execution and communication.
+> * By its design, Go proposes an approach for the construction of system software on multicore machines."
+
+## Why learn Go?
+
+* the syntax is easy to pick up - it feels simple and is easy to learn.
+* really nice documentation help with the above point - https://golang.org/doc/faq is a nice palce to start.
```
Comment (wlagrone) [inline _posts/2015-09-29-post-about-go.markdown]: palce ==> place

File: _posts/2015-09-29-post-about-go.markdown [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+* It has garbage collection - this is great for someone like me who is used to this being handled for me -  In many compiled languages one has to manage on your own, but Go does it for me!
```
Comment (wlagrone) [inline _posts/2015-09-29-post-about-go.markdown]: What makes this a reason to learn Go?  trying to think through the standard languages that wouldnt have it thus making Go a great choice.  Don't think you should remove this bullet point per se but maybe there is something about the GC approach that Go takes that makes it more efficient than other competitor languages

Comment (wlagrone) [inline _posts/2015-09-29-post-about-go.markdown]: as far as why go...  maybe add something about concurrency being a first class citizen.  that is something that gives is an interesting advantage over other languages

Comment (npupatk) [inline _posts/2015-09-29-post-about-go.markdown]: promagining => programming

**Comment by wlagrone** [comment]
:+1:

**Pull request merged**